.cico/cico_openshift_e2e.sh

```diff
@@ -7,24 +7,36 @@
 #
 # SPDX-License-Identifier: EPL-2.0
 #
-set -ex
+
+#!/usr/bin/env bash
+# exit immediately when a command fails
+set -e
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+# error on unset variables
+set -u
+# print each command before executing it
+set -x
 
 trap 'Catch_Finish $?' EXIT SIGINT
 
 # Catch the finish of the job and write logs in artifacts.
 function Catch_Finish() {
     # grab devworkspace-controller namespace events after running e2e
-    oc get events -n devworkspace-controller | tee ${ARTIFACTS_DIR}/devworkspace-controller-events.log
+    getDevWorkspaceOperatorLogs
 }
 
 # ENV used by PROW ci
 export CI="openshift"
 export ARTIFACTS_DIR="/tmp/artifacts"
+export NAMESPACE="devworkspace-controller"
 
 # Component is defined in Openshift CI job configuration. See: https://github.com/openshift/release/blob/master/ci-operator/config/devfile/devworkspace-operator/devfile-devworkspace-operator-master__v4.yaml#L8
 export CI_COMPONENT="devworkspace-operator"
-# OPENSHIFT_BUILD_NAMESPACE env var exposed by Openshift CI. More info about how images are builded in Openshift CI: https://github.com/openshift/ci-tools/blob/master/TEMPLATES.md#parameters-available-to-templates
-export IMG=default-route-openshift-image-registry.apps.build01.ci.devcluster.openshift.com/${OPENSHIFT_BUILD_NAMESPACE}/stable:${CI_COMPONENT}
+
+# DEVWORKSACE_OPERATOR env var exposed by Openshift CI in e2e test pod. More info about how images are builded in Openshift CI: https://github.com/openshift/ci-tools/blob/master/TEMPLATES.md#parameters-available-to-templates
+# Dependencies environment are defined here: https://github.com/openshift/release/blob/master/ci-operator/config/devfile/devworkspace-operator/devfile-devworkspace-operator-master__v5.yaml#L36-L38
+export IMG=${DEVWORKSACE_OPERATOR}
```

Comment:
Should this be `DEVWORKSPACE_OPERATOR`? instead of `DEVWORKSACE_OPERATOR` (missing a p)

Reply:
FacePalm... My mistake :( :( https://github.com/openshift/release/blob/master/ci-operator/config/devfile/devworkspace-operator/devfile-devworkspace-operator-master__v5.yaml#L36-L38
Need to change here